CHANGELOG.md

```diff
@@ -4,6 +4,16 @@ Notable changes.
 
 ## October 2022
 
+### [0.23.2]
```

Comment:
There are additionally some changes to the features test command that may be nice to call out

- https://github.com/devcontainers/cli/commit/ace8fa23ff2a9463e857bf5007359104dbb22ff7
- https://github.com/devcontainers/cli/commit/d9ee37741bc927a4378be5131b6b805eb1e392ab
- https://github.com/devcontainers/cli/commit/17ca5f90e5a45fb511e99dbd391bb82565776c26

Reply:
I'll mention them for 0.24.0. This is just a merge from the release/oct22 branch we used for stabilization.

Reply:
(I can add that in this PR.)